pipeline/scripts/update-electron-checksum.js

```diff
@@ -34,9 +33,10 @@ const writeLatestYAML = (latestPath, latest) => {
 };
 
 const updateElectronChecksum = async () => {
-    const latestPath = getLatestYAMLPath();
+    const parentDir = process.argv.length > 2 ? process.argv[2] : 'dist'; // TODO - refactor using library/envvar?
```

Comment:
`update-electron-checksum` needs to find the `latest.yml` files in 'dist' or 'signing-in-progress' to support both the current canary release & the new releases from this feature.